modules/order/src/Plugin/Field/FieldWidget/AdjustmentDefaultWidget.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+namespace Drupal\commerce_order\Plugin\Field\FieldWidget;
+
+use CommerceGuys\Intl\Formatter\NumberFormatterInterface;
+use Drupal\commerce_order\Adjustment;
+use Drupal\commerce_price\NumberFormatterFactoryInterface;
+use Drupal\commerce_price\Price;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\WidgetBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Plugin implementation of 'commerce_adjustment_default_widget'.
+ *
+ * @FieldWidget(
+ *   id = "commerce_adjustment_default_widget",
```

Comment:
The id should be "commerce_adjustment_default", we never add "_widget" to widget ids.